images/virtualization-artifact/pkg/controller/kvbuilder/kvvm_compare.go

```diff
@@ -205,6 +205,14 @@ func CompareKVVM(curr, next *KVVM) (*ChangeApplyActions, error) {
 		actions.Add(action)
 	}
 
+	{
+		action, err := CompareSysprep(curr, next)
```

Comment:
looks like Legacy. Now we don't use CompareKvvm. You should add sysprep to pkg vmchange.